Plugins/Time/Time.cpp

```diff
@@ -55,58 +55,44 @@ Time::~Time()
 
 ArgumentStack Time::GetTimeStamp(ArgumentStack&&)
 {
-    ArgumentStack stack;
-
     auto now = std::chrono::system_clock::now();
     auto dur = now.time_since_epoch();
 
     auto seconds = std::chrono::duration_cast<std::chrono::seconds>(dur).count();
 
-
-    Services::Events::InsertArgument(stack, (int)seconds);
-    return stack;
+    return Services::Events::Arguments((int)seconds);
 }
 
 ArgumentStack Time::GetHighResTimeStamp(ArgumentStack&&)
 {
-    ArgumentStack stack;
-
     auto now = std::chrono::system_clock::now();
     auto dur = now.time_since_epoch();
 
     auto count = std::chrono::duration_cast<std::chrono::microseconds>(dur).count();
 
-    Services::Events::InsertArgument(stack, (int32_t)(count / 1000000));
-    Services::Events::InsertArgument(stack, (int32_t)(count % 1000000));
-    return stack;
+    return Services::Events::Arguments((int32_t)(count % 1000000));
```

Comment:
This should return 2 values

`E [17:32:23] [NWNX_Core] Plugin 'NWNX_Time', event 'GetHighResTimeStamp': Tried to get a return value when one did not exist.` :(

Reply:
Ouch. Thanky.